src/components/Conversation.svelte

```diff
@@ -23,33 +22,30 @@ $: unread.count > 0 && setTimeout(() => saw(conversation.id), FIVE_SECONDS)
 
 async function accept() {
   try {
-    const response = await acceptCommittment(conversation.post.id)
-    post = response.updatePost
+    const { updatePost } = await acceptCommittment(post.id)
+    post = updatePost
   } catch (e) {
     // TODO: need errorhandling
   }
 }
 
-async function send() {
+async function sendMessage() {
   if (reply !== '') {
     let isNewConversation = !conversation.id
-    const response = await sendMessage(conversation.id, reply, post.id)
-    conversation.messages = response.createMessage.thread.messages
+    const updatedConversation = await send(reply, conversation)
     reply = ''
     
     if (isNewConversation) {
-      dispatch('new', {
-        id: response.createMessage.thread.id,
-        messages: response.createMessage.thread.messages
-      })
+      const dispatch = createEventDispatcher()
```

Comment:
I seem to remember the Svelte documentation saying that you need to instantiate the event dispatcher on the initial loading of the component, rather than later on... I'll see if I can find that documentation.